src/axis.js

```diff
@@ -1,5 +1,12 @@
-c3_chart_internal_fn.initAxis = function () {
-    var $$ = this, config = $$.config, main = $$.main;
+function _Axis(owner) {
```

Comment:
Can you remove `_`? Just `Axis` seems better from the uniformity point of view (e.g. Chart, ChartInternal and API).